models/bend_dao/ethereum/bend_dao_ethereum_schema.yml

```diff
@@ -27,7 +27,7 @@ models:
               - tx_hash
               - token_id
             check_columns:
-              - amount_original
+#              - amount_original
```

Comment:
CI may fail due to double typed comparison:
`select * from hive.test_schema.git_dunesql_0c0af3b_check_seed_bend_dao_ethereum_lending_amount_original__evt_type__borrower__block_number__tx_hash__token_id__r`